src/cfp/panel.njk

```diff
@@ -56,14 +56,12 @@ layout: base.njk
                 </ul>
             </div>
 
-            <div class="flex justify-center lg:justify-start pt-8">
```

Comment:
if you remove this, the button goes to the center. In larger screens, it looks off.